### DISCUSSION TITLE
Support for daisy chained MAX7219 seven segment displays

### DISCUSSION THREAD

**Discussion by hpagonis** [arendst/Tasmota#15316] [Ideas]
Hello developers,

I have three MAX7219 eight digit seven segment displays and I managed to daisy chain them together by altering the xdsp_15_tm1637.ino driver and the DisplayWidth command. I would like to contribute to the Tasmota project if you agree.

Documentation:
* [MAX7219 data sheet](https://www.openimpulse.com/blog/wp-content/uploads/wpsc/downloadables/MAX7219-Datasheet.pdf)

**Comment by barbudor** [comment]
Hi
A good PR is always welcome
This is how Tasmota works with the help of the community

You need to submit a PR through Github so it can be reviewed/tested

**Reply by teaalltr** [reply]
Hey @hpagonis any update on this?
Looks like the code is in these commits, https://github.com/hpagonis/Tasmota/commits/max7219-daisy-chain/
@arendst could it be rebased and merged? It would be a great feature to add 😄
